src/util/factory.ts

```diff
@@ -57,7 +57,13 @@ export namespace f {
 	}
 
 	export function number(value: number | string, flags?: ts.TokenFlags) {
-		return factory.createNumericLiteral(value, flags);
+		if (typeof value === "string") {
```

Comment:
The value should be converted into a number (to match TypeScript behavior), instead of creating a string literal.

Reply:
done!